src/filters/pixelate.ts

```diff
@@ -35,32 +41,40 @@ interface CenterOptions {
  * @returns pixelated Image
  */
 export function pixelate(image: Image, options: PixelateOptions): Image {
-  const { cellSize } = options;
+  const { cellSize, algorithm } = options;
   if (!Number.isInteger(cellSize)) {
     throw new TypeError('cellSize must be an integer');
   }
   if (cellSize < 2) {
     throw new RangeError('cellSize must be greater than 1');
   }
   const newImage = getOutputImage(image, options);
-
+  let value: number;
   for (let channel = 0; channel < image.channels; channel++) {
     for (let column = 0; column < image.width; column += cellSize) {
       for (let row = 0; row < image.height; row += cellSize) {
         const currentCellWidth = Math.min(cellSize, image.width - column);
         const currentCellHeight = Math.min(cellSize, image.height - row);
-
-        const center = getCenter({
+        let valuesOfSector = getArrayOfValues(image, channel, {
           width: currentCellWidth,
           height: currentCellHeight,
-          origin: {
-            column,
-            row,
-          },
+          origin: { column, row },
         });
-
-        const value = image.getValue(center.column, center.row, channel);
-
+        if (algorithm === 'mean') {
+          value = xMean(valuesOfSector);
```

Comment:
`xMean` is misleading because could indicate the mean is "directional". Maybe just call it `mean`? or `cellMean`?